src/Forms/Shared/Samples/Geometry/NearestVertex/readme.md

```diff
@@ -10,21 +10,28 @@ Determine the shortest distance between a location and the boundary of an area.
 
 ## How to use the sample
 
-Tap anywhere on the map. An orange cross will show at that location. A blue circle will show the polygon's nearest vertex to the point that was clicked. A red diamond will appear at the coordinate on the geometry that is nearest to the point that was clicked. If clicked inside the geometry, the red and orange markers will overlap. The information box showing distance between the clicked point and the nearest vertex/coordinate will be updated with every new location clicked.
+Tap anywhere on the map. An orange cross will show at that location. A blue circle will show the polygon's nearest vertex to the point that was tapped. A red diamond will appear at the coordinate on the geometry that is nearest to the point that was tapped. If tapped inside the geometry, the red and orange markers will overlap. The information box showing distance between the tapped point and the nearest vertex/coordinate will be updated with every new location tapped.
 
 ## How it works
 
 1. Get a `Geometry` and a `Point` to check the nearest vertex against.
 2. Call `GeometryEngine.NearestVertex(inputGeometry, point)`.
-3. Use the returned `ProximityResult` to get the `Point` representing the polygon vertex, and to determine the distance between that vertex and the clicked point.
+3. Use the returned `ProximityResult` to get the `Point` representing the polygon vertex, and to determine the distance between that vertex and the tapped point.
 4. Call `GeometryEngine.NearestCoordinate(inputGeometry, point)`.
-5. Use the returned `ProximityResult` to get the `Point` representing the coordinate on the polygon, and to determine the distance between that coordinate and the clicked point.
+5. Use the returned `ProximityResult` to get the `Point` representing the coordinate on the polygon, and to determine the distance between that coordinate and the tapped point.
 
 ## Relevant API
 
+* Geometry
 * GeometryEngine
 * ProximityResult
 
+## Additional information
+
+The value of `ProximityResult.distance` is planar (Euclidean) distance. Planar distances are only accurate for geometries that have a defined projected coordinate system, which preserves distance and length. The geometries in this sample are defined in Auxiliary Sphere (WKID 3857), which doesn't preserve distance. Hence we need to use the geodetic distance method to measure accurate distances in this example.
+
+Because the Web Mercator projected coordinate system is so common in web maps and it heavily distorts both area and length measurements, all geodetic measurement APIs are designed to allow geometries with Web Mercator spatial references as input. This allows you to measure geometries accurately in Web Mercator without having to project them to a different coordinate system.
```

Comment:
This is a bit confusing since the sample doesn't actually show any geodetic APIs. I like Java's code better for this reason:

> The value of `ProximityResult.distance` is planar (Euclidean) distance. Planar distances are only accurate for geometries that have a defined projected coordinate system, which maintain the desired level of accuracy. The example polygon in this sample is defined in California State Plane Coordinate System - Zone 5 (WKID 2229), which maintains accuracy near Southern California. Accuracy declines outside the state plane zone.